frontend/components/ResourceInfoCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Grid, Paper, Sx, Text, rem } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { ResourceInfo } from '@/util/types/fhir';
+import { ArrowNarrowRight } from 'tabler-icons-react';
+
+export interface ResourceInfoCardProps {
+  resourceInfo: ResourceInfo;
+  selected?: boolean;
+}
+
+export default function PatientInfoCard({ resourceInfo, selected }: ResourceInfoCardProps) {
+  return (
+    <Paper
+      shadow="sm"
+      p="md"
+      sx={theme => {
+        const style: Sx = {
+          ':hover': {
+            cursor: 'pointer',
+            backgroundColor: theme.colors.gray[0]
+          },
+          borderRadius: 12,
+          border: `${rem(1)} solid ${theme.colors.gray[3]}`
+        };
+        if (selected) {
+          style.border = `${rem(1)} solid ${theme.colors.blue[3]}`;
+        }
+        return style;
+      }}
+    >
+      <Grid align="center">
+        <Grid.Col span={10}>
+          <div>
+            <Text size="md" fw={700}>
+              {resourceInfo.id}
```

Comment:
I know we talked a bit about what would be best displayed here. I'm wondering if `name` would be a better identifier when available? Also, do we want version displayed here if available? @hossenlopp, I feel like I remember you having opinions during the mockups meeting.